extensions/txAdminClient/resource/sv_main.lua

```diff
@@ -113,6 +113,18 @@ function txaBroadcast(source, args)
     CancelEvent()
 end
 
+-- Announce message to all players
+RegisterCommand("txaAnnounce", function(source, args)
```

Comment:
The `RegisterCommand` should be in line 30-ish, and here just the function.  
Just like the other ones